privaterelay/pending_locales/en/pending.ftl

```diff
@@ -1,5 +1,17 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-# This is the Django equivalent of frontend/pendingTranslations.ftl
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This is the Django equivalent of frontend/pendingTranslations.ftl
+
+our-promise-header = Our promise to you
+our-promise-content = { -brand-name-relay } email masks forward to your email inbox any messages sent to that email mask—without revealing your real email address to the sender.
+what-can-you-do-with-relay-title = What can you do with { -brand-name-firefox-relay }?
+what-can-you-do-with-relay-point-one = Get emails without revealing your real email address to senders
+what-can-you-do-with-relay-point-one-subpoint-html = Just share an email mask instead. Email masks will forward emails to your inbox. You can create masks directly on { -brand-name-firefox }, with the <a href="{ $url }" { $attrs }>{ -brand-name-relay } extension</a>, or on your { -brand-name-relay } dashboard.

```

Comment:
suggestion (non-blocking): Make sure to add comments when you include variables, it helps translators have context. Or even if you're not using variables, if you think it needs additional context, add the comment. I personally have a hard time coming up with good names for the actual strings. Thanks!